atest/testdata/standard_libraries/string/get_matching_lines.robot

```diff
@@ -76,6 +100,11 @@ Test Get Lines Matching Pattern
     Should Be Equal    ${actual}    ${expected}
 
 Test Get Lines Matching Regexp
-    [Arguments]    ${input}    ${pattern}    ${expected}
-    ${actual} =    Get Lines Matching Regexp    ${input}    ${pattern}
+    [Arguments]    ${input}    ${pattern}    ${expected}    ${exact_match}=True
+    ${actual} =    Get Lines Matching Regexp    ${input}    ${pattern}    ${exact_match}
+    Should Be Equal    ${actual}    ${expected}
+
+Test Get Lines Containing Regexp
+    [Arguments]    ${input}    ${pattern}    ${expected}    ${exact_match}=False
+    ${actual} =    Get Lines Matching Regexp    ${input}    ${pattern}    ${exact_match}
     Should Be Equal    ${actual}    ${expected}
```

Comment:
`${exact_match}` argument to these keywords seems to unused. I'd actually prefer the first one not to pass `exact_match` at all to `Get Lines Matching Regexp` to verify that that is the default behavior. The latter should preferably use the named argument syntax (e.g. `exact_match=false`) to test the argument name.